app/views/content_items/consultation.html.erb

```diff
@@ -1,175 +1,193 @@
+      <div aria-labelledby="documents-title">
+        <%= render 'govuk_component/govspeak',
+            content: @content_item.documents,
+            direction: page_text_direction %>
+      </div>
+    <% end %>
   </div>
-<% end %>
+</div>
 
 <% if @content_item.ways_to_respond? %>
-  <div class="grid-row sidebar-with-body">
-    <div class="column-third">
+  <div class="grid-row">
+    <div class="column-two-thirds">
       <%= render 'components/heading', text: "Ways to respond", id: "ways-to-respond-title" %>
-    </div>
-    <div class="column-two-thirds" aria-labelledby="ways-to-respond-title">
-      <% @ways_to_respond_body = capture do %>
-        <% if @content_item.respond_online_url %>
-          <div class="call-to-action">
-            <p><%= link_to 'Respond online', @content_item.respond_online_url %></p>
-          </div>
-
-          <% if @content_item.email || @content_item.postal_address %>
-            <p>or</p>
+      <div aria-labelledby="ways-to-respond-title">
+        <% @ways_to_respond_body = capture do %>
+          <% if @content_item.respond_online_url %>
+            <div class="call-to-action">
+              <p><%= link_to 'Respond online', @content_item.respond_online_url %></p>
+            </div>
+
+            <% if @content_item.email || @content_item.postal_address %>
+              <p>or</p>
+            <% end %>
           <% end %>
-        <% end %>
 
-        <% if @content_item.response_form? %>
-          <p>
-            Complete a <%= link_to 'response form', @content_item.attachment_url %> and
-            <% if @content_item.email && @content_item.postal_address %>either<% end %>
-          </p>
-        <% end %>
+          <% if @content_item.response_form? %>
+            <p>
+              Complete a <%= link_to 'response form', @content_item.attachment_url %> and
+              <% if @content_item.email && @content_item.postal_address %>either<% end %>
+            </p>
+          <% end %>
 
-        <% if @content_item.email %>
-          <h3>Email to:</h3>
-          <p><%= mail_to @content_item.email, @content_item.email %></p>
-        <% end %>
 
-        <% if @content_item.postal_address %>
-          <h3>Write to:</h3>
-          <div class="contact">
-            <div class="content">
-              <%= simple_format(@content_item.postal_address) %>
+          <% if @content_item.response_form? %>
+            <p>
+              Complete a <%= link_to 'response form', @content_item.attachment_url %> and
+              <% if @content_item.email && @content_item.postal_address %>either<% end %>
+            </p>
+          <% end %>
+
+          <% if @content_item.email %>
+            <h3>Email to:</h3>
+            <p><%= mail_to @content_item.email, @content_item.email %></p>
+          <% end %>
+
+          <% if @content_item.postal_address %>
+            <h3>Write to:</h3>
+            <div class="contact">
```

Comment:
Looks like a potentially leaky class name. Would it make sense to update to BEM throughout? Or remove classes that aren't being used for styling?

Reply:
Same as above, this isn't really a change, it's a whitespace edit. I think it's ok to leave as-is for now.